rust.yaml

```diff
@@ -33,7 +33,6 @@ environment:
       - llvm15-dev
       - clang-15
       - openssl-dev
-      - libssh2-dev
```

Comment:
was it definitely `libssh2-dev`?  I couldn't see the duplicate

Reply:
Yeah it's here on line 30, just outside the preview.

<img width="266" alt="image" src="https://user-images.githubusercontent.com/1714486/210138582-4f50d08c-fcda-4ef2-9b30-7541c8187f05.png">

Reply:
doh I was looking at the whole file using the commit sha - i.e. with the duplicate removed